README.Rmd

```diff
@@ -67,7 +67,8 @@ With reference to the test data in this repo, you can run the `jitter` command l
 ```{bash}
 odjitter --od-csv-path data/od.csv \
   --zones-path data/zones.geojson \
-  --subpoints-path data/road_network.geojson \
+  --subpoints-origins-path data/road_network.geojson \
```

Comment:
I only updated this file; I'm having lots of issues regenerating the `.md`...

1) The bash commands all rely on an `odjitter` command existing in my `$PATH`, but if I'm developing that command, the new copy is in `./target/{release,debug}/odjitter` instead. I can temporarily change that, but that's a very clunky workflow. Or is the intention to do `cargo install` from my local development copy?
2) I had to `Rscript -e 'install.packages("od")'` and for `readr`, and also install pandoc. It very much weirds me out that these dependencies aren't more explicit, get installed in some system-wide place, and aren't pinned to a version. But we've discussed this philosophical difference before, it's fine. :)
3) When I regenerated the .md, there were a bunch of spurious diffs in the output with whitespace, and it left random `output_max50.geojson` files without any cleanup

Since there are more changes coming, I'm going to leave the Rmd and md out of sync and sort this out at the end.

Reply:
Thanks for the heads up and apologies for the clunky nature of it. There should be absolutely minimal dependencies. Plan: I can take care of the docs. Thinking of porting the .Rmd to .qmd which should remove the R dependency totally. Sound like a plan?

Reply:
Could be worth trying, but doesn't quarto turn around and use R or something else underneath for all the interactive script running? Why don't we just write a regular `.md` without any magic at all? Embedding command output in the README is an unusual practice for crates.io. Maybe the README could stay really narrowly focused, and the interactive walk-through style guide is better suited for a vignette of a future R package that interfaces with odjitter?

Reply:
> Could be worth trying, but doesn't quarto turn around and use R or something else underneath for all the interactive script running? Why don't we just write a regular `.md` without any magic at all? Embedding command output in the README is an unusual practice for crates.io. Maybe the README could stay really narrowly focused, and the interactive walk-through style guide is better suited for a vignette of a future R package that interfaces with odjitter?

I don't think so. Quarto does not depend on R (only if you try to compile to .pdf it by default tries to install a minimal LaTeX installation via R I think). Open minded and happy to look at reducing the length of the README although currently it provides context and usage about the `odjitter` crate not the (currently non existent) R/Python interfaces.